src/options.ts

```
@@ -69,7 +70,14 @@ document.addEventListener("DOMContentLoaded", async () => {
     chrome.storage.local.get("settings") as Promise<{ settings?: Settings }>,
   ]);
 
-  const settings: Settings = config.settings || {};
+  let rawSettings = config.settings;
```

ESLint

[error] 'rawSettings' is never reassigned. Use 'const' instead.

(prefer-const)